server/src/main/java/com/thoughtworks/go/server/newsecurity/filterchains/AuthorizeFilterChain.java

```diff
@@ -73,10 +73,13 @@ public AuthorizeFilterChain(AllowAllAccessFilter allowAllAccessFilter,
                 .addAuthorityFilterChain("/admin/materials/**", genericAccessDeniedHandler, ROLE_SUPERVISOR, ROLE_GROUP_SUPERVISOR)
                 .addAuthorityFilterChain("/admin/package_repositories/**", genericAccessDeniedHandler, ROLE_SUPERVISOR, ROLE_GROUP_SUPERVISOR)
                 .addAuthorityFilterChain("/admin/package_definitions/**", genericAccessDeniedHandler, ROLE_SUPERVISOR, ROLE_GROUP_SUPERVISOR)
-                .addAuthorityFilterChain("/admin/elastic_agent_configurations/**", genericAccessDeniedHandler, ROLE_SUPERVISOR, ROLE_GROUP_SUPERVISOR)
-                //allow /go/admin/environments page to be accessible by normal users
+
                 .addAuthorityFilterChain("/admin/environments/**", genericAccessDeniedHandler, ROLE_USER)
                 .addAuthorityFilterChain("/admin/config_repos/**", genericAccessDeniedHandler, ROLE_USER)
+                .addAuthorityFilterChain("/admin/elastic_agents/**", genericAccessDeniedHandler, ROLE_USER)
+                .addAuthorityFilterChain("/admin/admin/elastic_profiles/**", genericAccessDeniedHandler, ROLE_USER)
```

Comment:
these are redirects, refer [this](https://github.com/gocd/gocd/blob/master/server/src/main/webapp/WEB-INF/urlrewrite.xml#L133-L137) and [this](https://github.com/gocd/gocd/blob/master/server/src/main/webapp/WEB-INF/urlrewrite.xml#L617-L621) section of code.

Not whitelisting these URLs will cause the redirects to not work the page to throw a 403 for non-admin users.